chain-vote/src/tally.rs

```diff
@@ -116,6 +116,10 @@ impl EncryptedTally {
         }
     }
 
+    pub fn max_stake(&self) -> u64 {
```

Comment:
I don't think this is necessary and exposing this only for `EncryptedTallies` seems a bit strange to me